src/main/java/com/CezaryZal/api/body/BodySizeRepository.java

```diff
@@ -1,66 +1,45 @@
 package com.CezaryZal.api.body;
 
+import com.CezaryZal.api.body.entity.BodySize;
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 import org.springframework.stereotype.Repository;
-import org.springframework.transaction.annotation.Transactional;
 
-import javax.persistence.EntityManager;
-import javax.persistence.PersistenceContext;
-import javax.persistence.Query;
+import java.sql.Date;
 import java.time.LocalDate;
 import java.util.List;
+import java.util.Optional;
 
-@Transactional
 @Repository
-public class BodySizeRepository {
+public interface BodySizeRepository extends CrudRepository<BodySize, Long> {
```

Comment:
czemu nie JpaRepository? https://stackoverflow.com/questions/14014086/what-is-difference-between-crudrepository-and-jparepository-interfaces-in-spring

Reply:
Hyyy ok, dałem CRUDa bo nie porzebowałem wiecej opcji ale tak jak napisała na Stacku mogę śmiało dać JpaRepo